.rubocop.yml

```diff
@@ -12,3 +12,7 @@ Metrics/BlockLength:
 Style/DateTime:
   Exclude:
     - 'lib/snowplow_ruby_duid/cookie.rb'
+    
+Gemspec/RequiredRubyVersion:
+  Exclude:
+    - 'snowplow_ruby_duid.gemspec'
```

Comment:
shouldn't we add the Ruby version to the gemspec instead of ignoring this?

Reply:
No idea who is using this and there isn't any particular requirement.